test/cmdlineTests.sh

```diff
@@ -294,8 +291,21 @@ test_solc_behaviour "${0}" "ctx:=/some/remapping/target" "" "" 1 "" "Invalid rem
 printTask "Running general commandline tests..."
 (
     cd "$REPO_ROOT"/test/cmdlineTests/
-    for tdir in */
+    for tdir in ${*:-*/}
```

Comment:
Can you explain how this works?

Reply:
This is just `$*` to get all command line arguments but using the `${var}` syntax which lets me have a default (`${var:-default}`) of `*/` which kicks in when there are no arguments.

By the way, it will probably break if any dirs contain spaces. I originally used `$@` to get more reasonable quoting behavior but shellcheck complained about that and I did not try too hard. We do not use spaces there anyway and spaces in the absolute path to the test dir do not matter since the argument is supposed to be just the test dir name, not the whole path.

Reply:
Yep, let's just not use spaces...